lib/ClusteredMarker.js

```diff
@@ -19,6 +20,7 @@ const ClusteredMarker = ({
         longitude: geometry.coordinates[0],
         latitude: geometry.coordinates[1]
       }}
+      tracksViewChandges={tracksClusterViewChanges}
```

Comment:
typo here, wrong prop name: tracksViewChandges
It should be: tracksViewChanges={tracksClusterViewChanges}

Reply:
Damn, I'm lame at typos! Should be fixed